### PR TITLE
Fix typos found by codespell

### PR DESCRIPTION
**Pull request by lupchurchnull**
CLA: trivial

### CODE REVIEW

**Comment by tmshort** [comment]
I consider this trivial, but I defer to others if they think otherwise.

File: ssl/t1_trce.c [hunk 1]
```diff
@@ -1293,7 +1293,7 @@ static int ssl_print_certificate(BIO *bio, const SSL_CONNECTION *sc, int indent,
         x = NULL;
     }
     if (x ==  NULL)
-        BIO_puts(bio, "<UNPARSEABLE CERTIFICATE>\n");
```
[the remark this answers is not in the document]
Comment (t8m) [reply ssl/t1_trce.c:1296]: unparseable is also a possible alternative spelling of unparsable so there is no point in changing this.

[the remark this answers is not in the document]
Comment (lupchurchnull) [reply ssl/t1_trce.c]: I've reverted the change.

**Review by kroeckx — APPROVED** [review]

**Review by tom-cosgrove-arm — APPROVED** [review]
agree trivial, and annoyed I didn't spot these typos at the time!

**Review by t8m — APPROVED** [review]
OK with trivial

**Comment by t8m** [comment]
Merged to the master branch. Thank you for your contribution.

**Pull request closed without merging**